Code/RDGeneral/Dict.h

```diff
@@ -18,8 +18,8 @@
 #include <map>
 #include <string>
 #include <vector>
-#include <boost/any.hpp>
-#include <RDGeneral/Exceptions.h>
+#include "RDValue.h"
+#include "Exceptions.h"
```

Comment:
It is partly a habit.  I tend to use "Exceptions.h" to include files from the same package or header system since the header files will be searched from the current directory first.  Using "<>" never includes the current directory in the search path.  Basically, <RDGeneral/Exceptions.h> can only use the installed header file or needs to use the current include path and "Exceptions.h" uses the one in this directory.

It helps me know what the dependencies are at a glance, i.e. <RDGeneral/Exceptions.h> tells me that we are dependent on RDGeneral where "Exceptions.h" tells me that there is no external dependency.

Other than these subtle differences, I'm adaptable, and don't particularly care, It may just be a retraining of muscle memory.